app/controllers/SubmissionsController.scala

```diff
@@ -68,7 +67,7 @@ class SubmissionsController @Inject() (
         submissions <- customsDeclareExportsConnector.fetchSubmissions
         submissionsInDescOrder = submissions.sorted(Submission.newestEarlierOrdering)
       } yield Ok(submissionsPage(SubmissionsPagesElements(submissionsInDescOrder, submissionsPages)))
-        .removingFromSession(ExportsSessionKeys.declarationId)
+//        .removingFromSession(declarationId, errorFixModeSessionKey)
```

Comment:
Do you need this comment?

Reply:
:facepalm: I need line 70. I commented the line for testing something, but I forgot after to uncomment it.
Thanks for spotting this.